site/content/api/Command-And-Control-API.md

```diff
@@ -12,66 +12,50 @@ THIS IS A FIRST DRAFT VERSION OF THE API SPEC AND THUS STILL SUBJECT TO DISCUSSI
+| *correlation-id* | no        | *properties*             | *message-id* | MAY contain an ID used to correlate a response message to the original request. If set, it is used as the *correlation-id* property in the response, otherwise the value of the *message-id* property is used. |
+| *message-id*     | yes       | *properties*             | *string*     | MUST contain an identifier that uniquely identifies the message at the sender side. |
+| *reply-to*       | yes       | *properties*             | *string*     | MUST contain the source address that the client wants to received response messages from. This address MUST be the same as the source address used for establishing the client's receive link (see [Preconditions]({{< relref "#preconditions" >}})). |
 
-If a control message requires additional data to be properly executed (like command parameters), the body of the message MUST consist of a single AMQP *Data* section containing a control 
-message data. The format and encoding of the data is indicated by the *command*. It is up to the device to interpret the incoming body by taking `command` into 
-consideration.
+A command message MAY contain arbitrary payload to be sent to the device in an AMQP Data section. The value of the command message's *subject* value may provide a hint to the device regarding the format, encoding and semantics of the payload data.
 
-## Sending back control message execution result
+## Receiving the Command's execution result
 
-After execution of a control message on a device, device MUST send a response message indicating a status of that execution.
-
-**Preconditions**
-
-1. Client has established an AMQP connection with Hono.
-2. Client has established an AMQP link in role *sender* with Hono using target address `control-reply/${tenant_id}/{$device_id}`. Where `${device_id}` is an ID of the
-device that received the control message and `${tenant_id}` is the ID of the tenant the device belongs to.
-
-**Message Flow**
-
-*TODO* add sequence diagram
+After execution of a command on a device, the device MUST send a response message indicating the outcome of executing the command.
 
 **Message Format**
 
-The following table provides an overview of the properties a client needs to set on a *Control* message.
+The following table provides an overview of the properties set on a Command's response message.
 
 | Name | Mandatory | Location | Type | Description |
 | :--- | :-------: | :------- | :-------- | :---------- |
-| *correlation-id* | yes | *properties* | UTF-8 *string* | MUST contain the correlation ID used to match incoming control message with the response message containing result of execution. |
+| *correlation-id* | yes | *properties* | *string* | MUST contain the correlation ID used to match the command message with the response message containing the result of execution on the device. |
```

Comment:
in order to be consistent with the table for the request, the type should probably be *message-id*, right?